projects/storefrontlib/src/cms-components/checkout/config/default-checkout-config.ts

```diff
@@ -29,7 +29,7 @@ export const defaultCheckoutConfig: CheckoutConfig = {
         type: [CheckoutStepType.REVIEW_ORDER],
       },
     ],
-    express: false,
+    express: true,
```

Comment:
Maybe when working on WIP change config in storefrontapp instead. Just to avoid leaving it by accident.